app/src/main/res/values/strings.xml

```diff
@@ -360,6 +360,7 @@
   <string name="notifications">Notifications</string>
   <string name="display_nearby_notification">Display nearby notification</string>
   <string name="display_nearby_notification_summary">Tap here to see the nearest place that needs pictures</string>
+  <string name="tab_to_retry_nearby_notification">Some problems occured, tab to retry</string>
```

Comment:
Please replace this with "Unable to load nearest place that needs pictures", with the subtext underneath: "Tap to retry, or swipe to dismiss"